sdk/servicebus/Azure.Messaging.ServiceBus/src/Constants.cs

```diff
@@ -49,5 +49,12 @@ internal static class Constants
         public const int WellKnownPublicPortsLimit = 1023;
 
         public const string DefaultScope = "https://servicebus.azure.net/.default";
+
+        /// <summary>
+        /// The message appended to exceptions returned from the service that contains a link to the troubleshooting guide.
+        /// Usage errors with obvious causes do not contain this message.
+        /// </summary>
+        internal const string TroubleshootingMessage =
```

Comment:
Is there a reason this is `internal` and everything else in the class is `public`?

Reply:
Nope! will update for consistency.